From ecbc1826be931212ca60142ee09e6ce45cc62705 Mon Sep 17 00:00:00 2001
From: Werner Sembach <wse@tuxedocomputers.com>
Date: Thu, 24 Apr 2025 23:29:53 +0200
Subject: [PATCH] hwdb/keyboard: Map FN key on TUXEDO InfinityFlex 14 Gen1

Map the FN key of TUXEDO InfinityFlex 14 Gen1 that wrongly produces the F23
scancode by default.

Signed-off-by: Werner Sembach <wse@tuxedocomputers.com>
(cherry picked from commit d2502f55a2d9dc13f82cc1551a0639843fefb69e)

Resolves: RHEL-72701
---
 hwdb.d/60-keyboard.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 31664216fa..36bcdc9c3e 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -2070,6 +2070,10 @@ evdev:name:Quickstart Button 3:dmi:bvn*:bvr*:bd*:svnTOSHIBA*:pnPORTEGEZ830:*
 evdev:atkbd:dmi:*:svnTUXEDO:*:rvnNB02:*
  KEYBOARD_KEY_f8=fn
 
+# TUXEDO InfinityFlex 14 Gen1
+evdev:atkbd:dmi:*:svnTUXEDO:*:rnDN50Z-140HC-YD:*
+ KEYBOARD_KEY_6e=fn
+
 ###########################################################
 # VIA
 ###########################################################
